### ISSUE TITLE
Publish to Homebrew

### ISSUE THREAD

**Issue by panda2134** [creamlike1024/EasyLPAC#16]
It would make the installation on Macs easier if EasyLPAC is published to Homebrew.

**Comment by creamlike1024** [comment]
Good idea, but we need to have lpac submit to Homebrew first. In addition, it seems that Homebrew requires the cask application to have a valid signature to run on an Apple Silicon Mac with Gatekeeper enabled, which we currently do not have

**Comment by zhengbaizhen** [comment]
Those with letters will report an error
![Uploading 微信图片_20240909123711.png…]()

**Comment by zhengbaizhen** [comment]
ICCID
